statistics/table.go

```diff
@@ -276,35 +285,40 @@ func getPseudoRowCountByIndexRanges(sc *variable.StatementContext, indexRanges [
 	return totalCount, nil
 }
 
-func getPseudoRowCountByColumnRange(sc *variable.StatementContext, tableRowCount float64, ran types.ColumnRange) (float64, error) {
+func getPseudoRowCountByColumnRange(sc *variable.StatementContext, tableRowCount float64, columnRanges []*types.ColumnRange) (float64, error) {
```

Comment:
s/ByColumnRange/ByColumnRanges ?
Or you can add the function named  `getPseudoRowCountByColumnRanges`.